docs/conf.py

```diff
@@ -100,3 +100,21 @@
     "issue": ("https://github.com/fedora-infra/noggin/issues/%s", "#"),
     "pr": ("https://github.com/fedora-infra/noggin/pull/%s", "PR#"),
 }
+
```

Comment:
CI is unhappy because you need one more blank line here I think. Just run black on the file and you'll be all set